src/theme/variables.css

```diff
@@ -259,240 +259,3 @@
 .md {
 
 }
```

Comment:
```suggestion
}

```
Missing new line

Reply:
The file does have an ending newline now; the diff here just makes it hard to see.